# for some reason - {renv} does not like paths with these jupyter notebook
geo/natural-disasters-maxar-open-data/index.ipynb
geo/natural-disasters-maxar-open-data/notes
